operators/vtk_renderer/vtk_renderer.cpp

```diff
@@ -160,7 +160,7 @@ void VtkRendererOp::compute(InputContext& op_input, OutputContext&, ExecutionCon
   auto videostream = op_input.receive<gxf::Entity>("videostream").value();
   const auto videostream_tensor = videostream.get<Tensor>("");
   if (videostream_tensor) {
-    holoscan::gxf::GXFTensor in_tensor_gxf{videostream_tensor->dl_ctx()};
+    nvidia::gxf::Tensor in_tensor_gxf{videostream_tensor->dl_ctx()};
```

Comment:
Can you add the following up top like in a few other files so that we can detect if we are using GXF 4.0 (with DLPack support) or not?

```cpp

// If GXF has gxf/std/dlpack_utils.hpp it has DLPack support
#if __has_include("gxf/std/dlpack_utils.hpp")
  #define GXF_HAS_DLPACK_SUPPORT 1
  #include "gxf/std/tensor.hpp"
#else
  #define GXF_HAS_DLPACK_SUPPORT 0
  #include "holoscan/core/gxf/gxf_tensor.hpp"
#endif
```

Then we can modify the line here as follows to insert version-dependent code
```suggestion
  #if GXF_HAS_DLPACK_SUPPORT
    nvidia::gxf::Tensor in_tensor_gxf{videostream_tensor->dl_ctx()};
  #else
    holoscan::gxf::GXFTensor in_tensor_gxf{videostream_tensor->dl_ctx()};
  #endif
```

Reply:
I've added `operators/version_helper_macros.hpp` that can be reused by other operators.